src/main/rx/srxlv2_types.h

```diff
@@ -92,7 +92,7 @@ typedef enum {
 } srxlv2BindRequest;
 
 typedef enum {
-    NotBound = 0xAE,
+    NotBound = 0x0,
```

Comment:
Where did `0xAE` came from? Did I misinterpret it?

Reply:
You had it right based on the documentation, but this was one of the things Matt had to change when fixing the whole bind command implementation. I believe the change was related to some logic required to handle multi receiver systems.